cls/SourceControl/Git/Utils.cls

```diff
@@ -1230,7 +1230,7 @@ ClassMethod ImportItem(InternalName As %String, force As %Boolean = 0, verbose A
             if ($extract(InternalName, 1) = "/"){
                 set sc = ..ImportCSPFile(InternalName)
             } else{
-                set sc = $system.OBJ.Load(filename,"-l-d")
```

Comment:
Interesting that we were using -l - I wonder if there was a reason for this. I agree that displaying output is reasonable, for sure.

Question - what's the behavior if a single item fails to load/compile?

A better architecture here would be to maintain a list of things that were loaded by specific calls to ImportItem and then compile all of them at the same time after all have been loaded. As written this is likely to hit dependencies (especially on initial load); with compilation at the end it at least *should* be possible for a developer to fix compilation dependencies with the right values for System, CompileAfter, DependsOn, etc.

Reply:
Note from discussion today: just use PullEventHandler